EDEngineer/Resources/Data/entryData.json

```diff
@@ -1874,4 +1874,2476 @@
+    "Group": "Chemicals",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": [""]
+    },
+    {
+    "Name": "Weapon Component",
+    "ValueCr": 1000,
+    "BarterCost": 18,
+    "BarterValue": 9,
+    "FormattedName": "weaponcomponent",
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Tech",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": [""]
+    },
+    {
+    "Name": "Accident Logs",
+    "ValueCr": 4000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "accidentlogs",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": ["LAB", "PROC", "PWR", "AGRI"],
+    "ContainerType": ["Agricultural Data Port", "Data Port", "Industrial Data Port", "Laboratory Data Port", "Power Data Port"]
+    },
+    {
+    "Name": "Air Quality Reports",
+    "ValueCr": 3000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "airqualityreports",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Industrial Data Port", "Power Data Port"]
+    },
+    {
+    "Name": "Atmospheric Data",
+    "ValueCr": 4000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "atmosphericdata",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Laboratory Data Port"]
+    },
+    {
+    "Name": "Audio Logs",
+    "ValueCr": null,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "audiologs",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Security Data Port"]
+    },
+    {
+    "Name": "AX Combat Logs",
+    "ValueCr": null,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "axcombatlogs",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Security Data Port"]
+    },
+    {
+    "Name": "Ballistic Data",
+    "ValueCr": null,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "ballisticdata",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Security Data Port"]
+    },
+    {
+    "Name": "Biological Weapon Data",
+    "ValueCr": null,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "biologicalweapondata",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",

```

Comment:
`"Group": ",` is invalid

you can replace it with `"Group": "NO_GROUP"` as a placeholder

Reply:
Ah, that is a search and replace error. I will fix it.

Reply:
I used "Group" for "Item" and "Data" as suggested further down.